README.md

```diff
@@ -22,3 +22,72 @@ npm run push
 
 All staged code will be formatted using [prettier in a pre-commit
 hook](https://github.com/azz/pretty-quick#pre-commit-hook).
+
+
+## Development Process and Testing out Changes
+
+This process is still being fleshed out. There's some cowboying going on, but a safe route is listed below.
+
+For now, if you want to test changes on sheets, you are advised to:
+
+1. Check you have the latests changes from GitHub and the signup sheet. On a clean workspace:
+    ```
+        git checkout main
+        git pull
+        # Make sure things update smoothly
+        npm run pull
+        # Make sure there's no changes made - if so you'll need to figure out which is the right version - the version on GitHub or deployed on the spreadsheet. You can't move on to the next step until this is resolved
+    ```
+2. Create your branch for making changes. E.g.
+    ```
+        git checkout -b your-branch-name
+    ```
+3. First use the "Duplicate" button inside of Google Sheets to copy all the sheets you'll be affecting (mainly "Roles",
+   "Sign-Up Sheet", and "ToastmasterDetails")
+4. Next, update the `sheetnames.js` file to contain the new names, generally something like
+    ```javascript
+        // TODO(yourname): Return this back to normal before committing!
+        // const SIGNUP_SHEET_NAME = "Sign-up Sheet";
+        // const TM_DETAILS_SHEET_NAME = "ToastmasterDetails";
+        // const ROLES_SHEET_NAME = "Roles";
+        const SIGNUP_SHEET_NAME = "Copy of Sign-up Sheet";
+        const TM_DETAILS_SHEET_NAME = "Copy of ToastmasterDetails";
+        const ROLES_SHEET_NAME = "Copy of Roles";
+        const ROSTER_SHEET_NAME = "Roster";
+        const SIGNUP_TEMPLATE = "SignUp Template";
+    ```
```

Comment:
I'd go the route of `git add -p` and recommend not checking this in. I think this should be handled via an environment variable, but that would introduce some build steps...

Reply:
@acmiyaguchi Do you mind if I push that to a new GitHub issue to fix later? (or add on to #10)

I know it's lazy, but my main goal for today was actually to do #14 since I accidentally broke the date updating & to do #12.

I've only budgeted til 10pm to do them 😅 and don't think I'll be able to get to environment variables ( my goal was to do these during the conference, but I did some other bg tasks). Ideally we should have a bash script or something that can flip your environment variables too.

I suppose we could also commit sheetnames once and from then on put it in gitignore so you have to be really intentional to change it.

Reply:
Yes certainly, I wasn't suggesting to fix it here. The right way to do it (imo) would be to introduce a bundler like rollup (or webpack) and [replace strings like `process.env.SIGNUP_SHEET_NAME`](https://www.npmjs.com/package/@rollup/plugin-replace) + include a dotenv file.

Reply:
I'll add a link to this thread in #10